### ISSUE TITLE
Cannot find #[no_mangle] functions

### ISSUE THREAD

**Issue by gnzlbg** [gnzlbg/cargo-asm#13]
Currently `cargo asm` is unable to find `#[no_mangle]` functions.

**Comment by bbqsrc** [comment]
Any progress on this? :P

**Comment by gnzlbg** [comment]
Not really, but shouldn't be too hard to fix. In the code that gets a symbol name, and tries to demangle it, and see if it matches, we can just also try to see if the symbol name matches "as is".

**Comment by bbqsrc** [comment]
See https://github.com/gnzlbg/cargo-asm/pull/153. :smile:
